cmd/collector/app/grpcserver/grpc_server_test.go

```diff
@@ -38,19 +38,20 @@ func TestFailToListen(t *testing.T) {
 	l, _ := zap.NewDevelopment()
 	handler := app.NewGRPCHandler(l, &mockSpanProcessor{})
 	server := grpc.NewServer()
-	addr, err := StartGRPCCollector(1, server, handler, &mockSamplingStore{}, l, func(e error) {
+	const port = -1 // Invalid port
```

Comment:
code as documentation: `const invalidPort = -1`

Reply:
Lol I was thinking this as I submitted the commit. Will update.